service/src/main/java/uk/gov/hmcts/reform/fpl/service/ApplicantMigrationService.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Service
+public class ApplicantMigrationService {
```

Comment:
I commented above - this could be a generic SetMigratedService.

Reply:
True, just matter of finding best time and place to make it generic. Second PR is the latest we should do it.